help/gmcommands.txt

```diff
@@ -0,0 +1,123 @@
+ <- @@index|<<Back to Index>>@@
```

Comment:
Add a note to the effect "this is a static copy of the list and may be outdated, see `@help` for what this server actually supports".

Also mention the other forms of `@help`